guidelines/chair-group.md

```diff
@@ -0,0 +1,35 @@
+## Chair group
+
+#### Definition and Motivation
+
+The Chair Group is a representative body of the Message Format WG that consists in spreading of the WG responsibilities among WG members that volunteered to get involved in the group chairing activities. Being part of a chair group requires an additional commitment that comes with a set of duties/responsibilities.
+
+#### Responsibilities
+
+- Ensure that the composition of the WG is well balanced in order that a high number of representatives are involved in the work of WG.
+
+- Encourage the participants, attending the meeting and to collaborate actively.
+
+- Manage and accept WG applications.
+
+- Manage and organize all WG communication channels, email, mail group, boards and Github repositories
+  
+- Prioritize, label and organize the tasks of the WG.
+
+- Use the possibility of e-mail or  synchronization meetings to discuss issues or organize WG work.
+
+- Create and update events and agenda.
+
+- Participate in activities to share and promote the WG and all possible synergies that could improve the overall result
+
+- Take care of the minutes of the meetings, including recording of all decisions. The minutes and presentations given in WG should be available in the WG Drive
+
+- Participate and on the preparation of technical drafts or documentation.
+
+
+#### Participate
+
+Share your disposal to participate using one of the following using one of the following channels:
```

Comment:
Change "disposal" to "interest".  Also, +1 to Shane's comment on having a different email address for that (not the main list).